docker/oso-monitor-sso/src/start.sh

```diff
@@ -22,8 +22,8 @@ ansible-playbook /root/config.yml
 echo
 echo 'Running SSO functionality check every 24 hours'
 echo '----------------'
-/usr/local/bin/ops-run-in-loop 86400 "ops-runner -f -s 15 -n check.sso.iam.status /usr/local/bin/check_sso_service.py &>> /var/log/monitor-sso.log'" &
+/usr/local/bin/ops-run-in-loop 86400 ops-runner -f -s 15 -n check.sso.iam.status /usr/local/bin/check_sso_service &>> /var/log/monitor-sso.log &
```

Comment:
On this one and the one below it, because I changed the ops-run-in-loop script not to use eval, it makes it necessary to pass the command and args as a command and separate args, so quotes removed. Also, I renamed all the python scripts so that when the RPM installs them, they don't have a .py extension.